### ISSUE TITLE
Does 2.1.0 still support C++11

### ISSUE THREAD

**Issue by grunk** [vincentlaucsb/csv-parser#141]
Hello,
I was using csv-parser 1.3.3 whithout any problem.

I tried to uprgrade to 2.1.0 (mostly because of #105 ) but it does not compile on VS 2019 with a project configured to use C++11 (it have to build on old Gcc version too , that's why)

I have the following errors : 

> C2039	'get_string_view': is not a member of 'csv::CSVRow' parser\csv.cpp	44	

Which is relative to this code (was working fine previously) : 
```
for (csv::CSVRow& row : reader) {
    resultFile << row.get_string_view(0) << "," << row.get_string_view(29) << std::endl;
}
```
Maybe the API has changed ?

And also 

> C2429	language feature 'terse static assert' requires compiler flag '/std:c++17' (compiling source file parser\csv.cpp)	lapi	D:\C++\Projets\v8\lib\MSVC\x64\VC142\csv-parser\single_include\csv.hpp	4809

Which seems related to C++11 specifically
	
Thanks for your help

**Comment by pedrodaniel10** [comment]
From what I've tried, versions 2.x.x including beta cannot be compiled with C++11.

**Comment by xgdgsc** [comment]
Obviously not working with c++11 and 14 https://github.com/vincentlaucsb/csv-parser/issues/176

**Comment by vincentlaucsb** [comment]
Should be fixed by #177
